src/components/Event.vue

```diff
@@ -30,9 +30,9 @@
         </p>
 
         <p class="description">
-          Description. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce sed lobortis erat, a hendrerit mauris. Proin nec turpis nunc. Fusce risus orci, ornare vel dictum eget, volutpat non purus. Donec vitae varius.
+          {{ event.properties.type.subtype.description }}
         </p>
-        <p><a href="#">Permit # {{ event.properties.id }}</a></p>
```

Comment:
can you add a TODO or FIXME in here to what you want this to link to?

Reply:
👍 good call

Reply:
9cae9c2